tests/cities.test.js

```diff
@@ -0,0 +1,97 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const Cities = require('../src/models/cities');
+const mongoose = require('mongoose');
+const {server} = require('../src/app');
+chai.should();
+chai.use(chaiHttp);
+
+describe("GET Tests Cities",()=>{
+    it("GET /Cities", (done) =>{
+       chai.request(server).get("/cities/61645a187cab4c38442c4ed6")
+        .end( (error, response) => {
+            response.should.have.status(200);
+            response.body.should.be.a("array");
+            response.body.length.should.be.eq(1);
+            done();
+        });
+    });
+
+    it("GET /Cities Not Found", (done) =>{
+        chai.request(server).get("/cities/61645a187cab4c38442c4ed7")
+        .end( (error, response) => {
+            response.should.have.status(404);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("No se encontro registro(s)");
+            done();
+        });
+    });
+
+    it("GET /Cities Bad Request", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    it("GET /Cities Bad Request", (done) => {
```